source/language/gates.rst

```diff
@@ -338,15 +342,12 @@ respectively.
 
    // A reversible boolean function
    // Demonstrates use of ``ctrl(n) @`` and ``negctrl(n) @``
-   qubit[5] a;
-   qubit f;
-   reset f;
-   negctrl(2) @ ctrl(3) @ x a, f;
-   negctrl(2) @ ctrl(2) @ x a[0], a[3], a[1], a[2], f;
-   negctrl @ ctrl(3) @ x a[0], a[1], a[3], a[4], f;
-   negctrl @ ctrl(3) @ x a[1], a[0], a[3], a[4], f;
-   ctrl(3) @ x a[0], a[1], a[2], f;
-   negctrl(3) @ ctrl @ x a[0], a[1], a[2], a[3], f;
+  qubit[3] a;
+  qubit[3] b;
+  ctrl(3) @ x a[1], a[0], a[2], b[2];
+  negctrl (3) @ ctrl @ x a[0], b[1], a[2], b[0], b[2];
+  negctrl @ ctrl(2) @ negctrl @ x a[0], b[0], a[2], a[1], b[2];
+  negctrl (2) @ ctrl @ x b[1], a, b[0], b[2];
```

Comment:
The indentation here has got broken - I think a space got removed.  The new set of examples looks fine, though I'd keep the non-register `qubit f` in the mix just for completeness.